src/python/pants/init/logging.py

```diff
@@ -135,9 +136,16 @@ def setup_logging(global_bootstrap_options: OptionValueContainer, stderr_logging
         global_level, log_show_rust_3rdparty, use_color, show_target, log_levels_by_target
     )
 
-    print_exception_stacktrace = global_bootstrap_options.print_exception_stacktrace
+    print_stacktrace = resolve_conflicting_options(
```

Comment:
While this duplicates `engine_initializer.py`, I didn't think it's worth factoring up because this code will be deleted soon.

Reply:
Deleted soon because of @gshuflin's work on the rust client? Or something else?

Reply:
I mean deleted once the deprecation cycle is done.